composer.json

```diff
@@ -41,7 +41,7 @@
         "doctrine/doctrine-bundle": "^1.10 || ^2.0",
         "doctrine/doctrine-fixtures-bundle": "^2.4 || ^3.0",
         "doctrine/inflector": "^1.4.1 || ^2.0.1",
-        "doctrine/orm": "^2.5.3",
+        "doctrine/orm": "^2.6.1",
```

Comment:
The lowest version we get is `2.6.0` but that has an error in it with combined IDs like used in FormatOptions: https://github.com/sulu/sulu/blob/08adb07245d7b125496f47894bc1b013bb10dc30/src/Sulu/Bundle/MediaBundle/Resources/config/doctrine/FormatOptions.orm.xml#L9-L10 

So required atleast `^2.6.1`